src/components/dashboard/sidebar/DashboardSidebar.jsx

```diff
@@ -20,32 +32,49 @@ class DashboardSidebar extends React.Component {
       );
     }
     return message;
-  }
-
-  render() {
-    return (
-      <>
-        <SidebarBlock
-          title="Need help?"
-          titleOptions={{ tag: 'h3', className: 'h4' }}
-          className="mb-5"
-        >
-          <p>
-            For technical support, visit the{' '}
-            <a className="text-underline" href="https://support.edx.org/hc/en-us">edX Help Center</a>.
-          </p>
-          <p>
-            To request more benefits or specific courses, {this.renderContactHelpText()}.
-          </p>
-        </SidebarBlock>
-        <SidebarBlock>
-          <OfferSummaryCard />
-        </SidebarBlock>
-      </>
-    );
-  }
-}
+  };
 
-DashboardSidebar.contextType = AppContext;
+  return (
+    <>
+      <SidebarCard
+        title={CATALOG_ACCESS_CARD_TITLE}
+        cardClassNames="border-primary catalog-access-card mb-5"
+        titleClassNames="mb-3"
+      >
+        { subscriptionPlan && (
+          <SubscriptionSummaryCard
+            subscriptionPlan={subscriptionPlan}
+            className="mb-3"
+          />
+        )}
+        { offers && (
```

Comment:
What is `offers` here? I believe its value is actually an object like:

```js
{
  loading: false,
  offers: [],
  offersCount: 0,
  error: null,
}
```

which will always evaluate as `true`.

Reply:
yeah good point, the reason this still works is because of the check for `offersCount>0` in `OfferSummaryCard`. Since I don't need to provide anything but `offersCount` to `OfferSidebarCard` I'll change this to have `offersCount` as the required prop and check for `offersCount>0` in `DashboardSidebar`